pkg/types/types.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	"context"
+	"time"
+)
+
+type Task struct {
+	UserID, ChatID int64
```

Comment:
```suggestion
	ID, UserID, ChatID int64
```

Надо вернуть всё-таки ID. Я попутала. Этот ID надо будет обозначить в CreateTask. То есть ты сначала делаешь t.ID = ms.nextID, а потом уже всё остальное. Таким образом у тебя в ключе будет этот айдишник и в самой структуре, потому что нам он там понадобится